### PR TITLE
Add an extra command to the troubleshooting guide for react-native-typescript

### PR DESCRIPTION
**Pull request by DibyajyotiMishra**
Add troubleshooting guide for react native typescript

### CODE REVIEW

**Comment by Simek** [comment]
Refs: #2952

**Comment by cortinico** [comment]
Makes sense to have inside the note here 👍 Should we use a `:::note` admonitions here? https://docusaurus.io/docs/next/markdown-features/admonitions

@DibyajyotiMishra Also you have a prettier failure on the CI. Can you take a look?

**Review by cortinico — APPROVED** [review]

**Comment by DibyajyotiMishra** [comment]
> Makes sense to have inside the note here 👍 Should we use a `:::note` admonitions here? https://docusaurus.io/docs/next/markdown-features/admonitions
> 
> @DibyajyotiMishra Also you have a prettier failure on the CI. Can you take a look?

yeah sure.

**Comment by DibyajyotiMishra** [comment]
@Simek @cortinico I have fixed all the linting issues. Can you please review it?

**Review by cortinico — COMMENTED** [review]
Please udpate the whole block to use the `:::note` admonition:

```
:::note

If the above command is failing, you may have an old version of `react-native` or `react-native-cli` installed globally on your system. To fix the issue try uninstalling the CLI:

- `npm uninstall -g react-native-cli` or `yarn global remove react-native-cli`

and then run the `npx` command again.
Optionally, you can also use the command given below to get started with your template.

- `npx react-native --ignore-existing init MyApp --template react-native-template-typescript`
:::
```

**Comment by DibyajyotiMishra** [comment]
> Please udpate the whole block to use the `:::note` admonition:
> 
> ```
> :::note
> 
> If the above command is failing, you may have an old version of `react-native` or `react-native-cli` installed globally on your system. To fix the issue try uninstalling the CLI:
> 
> - `npm uninstall -g react-native-cli` or `yarn global remove react-native-cli`
> 
> and then run the `npx` command again.
> Optionally, you can also use the command given below to get started with your template.
> 
> - `npx react-native --ignore-existing init MyApp --template react-native-template-typescript`
> :::
> ```

Done.  ✅

**Comment by DibyajyotiMishra** [comment]
> Please udpate the whole block to use the `:::note` admonition:
> 
> ```
> :::note
> 
> If the above command is failing, you may have an old version of `react-native` or `react-native-cli` installed globally on your system. To fix the issue try uninstalling the CLI:
> 
> - `npm uninstall -g react-native-cli` or `yarn global remove react-native-cli`
> 
> and then run the `npx` command again.
> Optionally, you can also use the command given below to get started with your template.
> 
> - `npx react-native --ignore-existing init MyApp --template react-native-template-typescript`
> :::
> ```

@Simek can you please review this?

File: docs/typescript.md [hunk 1]
```diff
@@ -17,15 +17,17 @@ You can use the [TypeScript template][ts-template]:
...
+
+If the above command is failing, you may have an old version of `react-native` or `react-native-cli` installed globally on your system. To fix the issue try uninstalling the CLI:
+
+- `npm uninstall -g react-native-cli` or `yarn global remove react-native-cli`
+
+and then run the `npx` command again.
+Optionally, you can also use the command given below to get started with your template.
+
+- `npx react-native --ignore-existing init MyApp --template react-native-template-typescript`
+  :::
```
Comment (cortinico) [inline docs/typescript.md:30]:
```suggestion

:::
```

**Comment by DibyajyotiMishra** [comment]
@Simek is there anything which is stopping this merge? Please let me know so that I can fix it.

**Review by cortinico — APPROVED** [review]

**Comment by DibyajyotiMishra** [comment]
Thanks a lot @Simek and @cortinico for all your support and cooperation through out. :)

**Pull request merged**